### ISSUE TITLE
Intermittently long start up times for sample-apiserver.

### ISSUE THREAD

**Issue by cheftako** [kubernetes/kubernetes#65239]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
Related to issue #63622 and fix #64994.
Looking https://k8s-testgrid.appspot.com/sig-api-machinery#gce we seem some long start up times for sample-apiserver in the e2e test.
Looking at the last 20 runs I see -
Test Time | Sample API-Server Startup Time | Result
2018-06-18 00:18 PDT | 9s  |
2018-06-18 00:55 PDT | 21s | 
2018-06-18 01:33 PDT | 15s | 
2018-06-18 02:12 PDT | 15s | 
2018-06-18 02:49 PDT | 7s  | 
2018-06-18 03:26 PDT | 9s  | 
2018-06-18 04:04 PDT | 12s | 
2018-06-18 04:42 PDT | 11s | 
2018-06-18 05:19 PDT | 16s | 
2018-06-18 05:59 PDT | 11s | 
2018-06-18 06:36 PDT | 11s | 
2018-06-18 07:13 PDT | 9s  |
2018-06-18 07:50 PDT | 11s | 
2018-06-18 08:28 PDT | 28s |
2018-06-18 09:07 PDT | 15s |
2018-06-18 09:45 PDT | 61s | (FAILED)
2018-06-18 10:29 PDT | 12s |
2018-06-18 11:08 PDT | 9s  |
2018-06-18 11:47 PDT | 16s |
2018-06-18 12:26 PDT | 11s |
2018-06-18 13:03 PDT | 8s  |
2018-06-18 13:43 PDT | 21s |
2018-06-18 14:23 PDT | 12s | 
2018-06-18 15:00 PDT | 11s |
2018-06-18 15:41 PDT | 11s |
2018-06-18 16:19 PDT | 16s | 
2018-06-18 16:58 PDT | 13s |

**What you expected to happen**:
I would expect the reference server to start in a consistently quick amount of time. (< 10s)

**How to reproduce it (as minimally and precisely as possible)**:
Look the the CI.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by liggitt** [comment]
anything interesting in the log of the long startups?

**Comment by liggitt** [comment]
I don't think it would affect anything, but https://github.com/kubernetes/kubernetes/pull/64993 should resolve the one error we were seeing in the extension apiserver startup path

**Comment by cheftako** [comment]
Sadly the sample-apiserver logs are essentially deleted from the CI machines once the test completes. I did not see anything obviously wrong in kube-apiserver logs.
